web-extensions/chrome/engine/bookmycomics.js

```diff
@@ -51,9 +51,31 @@ function BmcEngine(hostOrigin, readerName, comicInfo) {
                                                                       'err': err.message}));
                 }
                 this._ui.toggleSidePanel();
-                notifyResult('Alias Comic', retErr);
+                this._ui.makeNotification('Alias Comic', retErr);
             });
         });
+    // Handle "Delete"
+    this._messaging.addWindowHandler(
+        ENGINE_ID,
+        evData => evData.type === 'action' && evData.action === 'delete',
+        evData => {
+            this.delete(
+                evData.comic.id,
+                (evData.source || {}).reader,
+                (evData.source || {}).name,
+                err => {
+                    let retErr = null;
+                    if (err) {
+                        retErr = new Error(LOCALIZATION.getString('S46', {
```

Comment:
`LOCALIZATION.getString` evolved a bit. You want to show an error so you should use `LOGS.getString` instead.

Reply:
I'll look into it
 - [ ] Update usages of `LOCALIZATION.getString()`